cmd/gateway/commands.go

```diff
@@ -377,3 +384,29 @@ func createSleepCommand() *cobra.Command {
 
 	return cmd
 }
+
+func parseFlagKeysAndValues(flags *pflag.FlagSet) (flagKeys, flagValues []string) {
+	flagKeys = []string{}
+	flagValues = []string{}
+
+	flags.VisitAll(
+		func(flag *pflag.Flag) {
+			flagKeys = append(flagKeys, flag.Name)
+			switch flag.Value.Type() {
```

Comment:
Would it be of better style just to use an `if` check here for booleans and remove the switch statement? Had this in the case where we want to report int values too and thus would have another case for "int", but seems like that is not in the plan currently.

Reply:
this is how it will look:
```go
			if flag.Value.Type() == "bool" {
				flagValues = append(flagValues, flag.Value.String())
			} else {
				var val string
				if flag.Value.String() == flag.DefValue {
					val = "default"
				} else {
					val = "user-defined"
				}

				flagValues = append(flagValues, val)
			}
```

to me, this way looks clearer - we clearly see there are only two cases we care about